pkg/imgpkg/cmd/copy_repo_src_test.go

```diff
@@ -534,9 +534,14 @@ func TestToRepoBundleCreatesValidLocationOCI(t *testing.T) {
 	fakeRegistry := helpers.NewFakeRegistry(t, logger)
 	defer fakeRegistry.CleanUp()
 
+	dockerhubProxy := "index.docker.io"
```

Comment:
This logic looks like it should be moved to `test/helpers/env.go` this is the struct that handles all the environment variable setup

Reply:
I have moved that to env.go as functions rather than methods of Env so as not to have to build an instance of Env in copy_repo_src_test.go.